### PR TITLE
cninja: fix license installation dir

### PR DESCRIPTION
**Pull request by raedrizqie**

### CODE REVIEW

File: mingw-w64-cninja/PKGBUILD [hunk 1]
```diff
@@ -4,18 +4,18 @@ _realname=cninja
...
 url="https://github.com/jcelerier/cninja"
-license=('spdx:AGPL-3.o-or-later')
+license=('spdx:AGPL-3.0-or-later')
 depends=("${MINGW_PACKAGE_PREFIX}-cmake"
          "${MINGW_PACKAGE_PREFIX}-clang"
          "${MINGW_PACKAGE_PREFIX}-lld"
          "${MINGW_PACKAGE_PREFIX}-ninja"
          "${MINGW_PACKAGE_PREFIX}-libc++")
-makedepends=("${MINGW_PACKAGE_PREFIX}-boost" "${MINGW_PACKAGE_PREFIX}-cc")
+makedepends=("${MINGW_PACKAGE_PREFIX}-boost")
```
Comment (mmuetzel) [inline mingw-w64-cninja/PKGBUILD:18]: If I read the build log correctly, this package needs a C++ compiler to be built.
Why did you remove it from `makedepends`?

Comment (raedrizqie) [reply mingw-w64-cninja/PKGBUILD]: i was following the aur package https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=cninja

**Comment by ognevny** [comment]
I guess it's fine now
